seedfarmer/__init__.py

```diff
@@ -74,3 +74,4 @@ def configure(configuration: CodeSeederConfig) -> None:
             'timeout 15 sh -c "until docker info; do echo .; sleep 1; done"',
         ]
         configuration.python_modules = [f"seed-farmer=={__version__}"]
+        configuration.runtime_versions = {"nodejs": "14", "python": "3.9", "docker": "20"}
```

Comment:
have we confirmed this? apparently our the base codebuild image doesn't support setting a docker runtime.

Reply:
The initial tests worked, but the docker runtime will be removed to avoid confusion